### PR TITLE
Remove unused metrics

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

We were generating some metrics that were totally unused on the frontend. Lets save some work and get rid of them! Get em' outta here!   

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by michielderoos** [comment]
[Remove unused metrics](https://trello.com/c/Rm2XroV9/666-remove-unused-metrics)

**Review by tristanhcole — COMMENTED** [review]
Tested 👍 ... one small comment otherwise looks good to me.

File: app/client/reducers/metric/types.ts [hunk 1]
```diff
@@ -30,19 +30,10 @@ export interface DailyVolume {
...
   master_wallet_balance: number;
-  total_beneficiaries: number;
   total_distributed: number;
   total_withdrawn: number;
   total_reclaimed: number;
-  total_exchanged: number;
-  total_spent: number;
-  total_users: number;
-  transfer_use_breakdown: object;
-  zero_balance_count: number;
```
[the remark this answers is not in the document]
Comment (michielderoos) [reply app/client/reducers/metric/types.ts:47]: The new metrics code has/had all of the metrics the old dashboard used in addition to the new ones. A lot of the work on new metrics happened before we changed the shape of the API output (and before any FE work was done), so some of old metrics + new metrics coexisted in the API for backwards compatibility for a while.

File: app/server/utils/metrics/participant_stats.py [hunk 2]
```diff
@@ -19,15 +19,6 @@ def __init__(self, group_strategy, timeseries_unit = 'day'):
...
-            metric_name='total_beneficiaries',
-            query=total_beneficiaries_query,
-            object_model=User,
-            stock_filters=[filters.beneficiary_filters],
-            caching_combinatory_strategy=metrics_cache.COUNT,
-            filterable_by=self.filterable_attributes))
-
         total_vendors_query = db.session.query(User)
         self.metrics.append(metric.Metric(
             metric_name='total_vendors',
```
[the remark this answers is not in the document]
Comment (michielderoos) [reply app/server/utils/metrics/participant_stats.py]: Done!

**Review by tristanhcole — APPROVED** [review]
LGTM! 📊

File: app/client/reducers/metric/types.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tristanhcole) [reply app/client/reducers/metric/types.ts:47]: That makes total sense!

**Pull request merged**